AuthenticationAndAuthorization/AuthenticationAndAuthorization/Role/UserConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace AuthenticationAndAuthorization.Role
+{
+    public static class UserConstants
+    {
+        public const string Admin = "Admin";
+        public const string User = "User";
+    }
```

Comment:
I suggest creating IReadOnlyList static property and then use it while adding roles. This way you don't create new arrays each time and you have a single source of roles in your app